tests/operators/test_filters.py

```diff
@@ -179,6 +179,24 @@ def test_generate_mask_fail_wrong_condition(cube):
         filters.generate_mask(cube, "!<", 10)
 
 
+def test_generate_mask_rename(cube):
+    """Generates a mask and checks rename."""
+    mask = cube.copy()
+    mask.data[:] = 0
+    mask.data[cube.data == 276] = 1
+    mask.rename(f"mask_for_{cube.name()}_==_276")
+    assert mask.name() == filters.generate_mask(cube, "==", 276).name()
```

Comment:
```suggestion
    expected = f"mask_for_{cube.name()}_==_276"
    assert filters.generate_mask(cube, "==", 276).name() == expected
```
As we are just testing the name here, there is no need to construct a full cube to test against.